frontend/src/pages/Cart/index.tsx

```diff
@@ -1,3 +1,87 @@
 import { h } from 'preact';
+import { useEffect, useState } from 'preact/hooks';
+import { CartService } from '../../services/cartService';
 
-export const Cart = () => <p>Cart</p>;
+export const Cart = () => {
+  const [cart, setCart] = useState(CartService.getItems());
+  const [quantityProducts, setQuantityProducts] = useState(null);
+  const [totalPrice, setTotalPrice] = useState(0);
+
+  useEffect(() => {
+    CartService.watch(setCart);
+  }, []);
+
+  useEffect(() => {
+    setTotalPrice(
+      cart.reduce((sum, i) => sum + i.price * (i.quantity || 1), 0),
```

Comment:
don't use short variable names

Reply:
for this case will be preferable to use useMemo, instead of useEffect and useState